src/components/cesium_map/elements/navigationButton.js

```diff
@@ -29,18 +30,20 @@ const InformationButton = () => {
     };
   };
 
+  const facet = props.facet || source;
+
   return (
-    <span className="cesium-navigationHelpButton-wrapper">
+    <>
       <button className="cesium-button cesium-toolbar-button cesium-navigation-help-button"
         onClick={() => toggle()}>
         <img src={info} alt="informatioin icon"></img>
       </button>
       <div id="legend"
         className="cesium-navigation-help"
         style={{ minWidth: "200px" }}>
-        <div className="cesium-click-navigation-help-visible">
-          <table className="cesium-navigation-help-instructions">
-            <tbody>
+        <div className="cesium-click-navigation-help-visible" >
+          <table className="cesium-navigation-help-instructions" >
+            <tbody style={{ height: "400px", overflow: "auto", display: "block", border: "white solid 1px" }}>
```

Comment:
In the future, we might have many colors in the legend. This style makes the popup window scrollable.